### ISSUE TITLE
Kite installation error

### ISSUE THREAD

**Issue by Abdullah-B** [spyder-ide/spyder#14801]
### What steps will reproduce the problem?

when i choose to install KITE i immedietly get this

### Traceback
```python-traceback
  File "D:\AbdullahAli\Anaconda\lib\site-packages\spyder\plugins\completion\kite\widgets\install.py", line 308, in install
    self.setup(integration=False, welcome=False, installation=True)
TypeError: setup() got an unexpected keyword argument 'welcome'
WARNING:spyder.widgets.github.backend:Failed to send bug report on Github. response={'code': 404, 'json': {'message': 'Not Found', 'documentation_url': 'https://docs.github.com/rest/reference/issues#create-an-issue'}}
```

## Versions

* Spyder version: 4.2.2 
* Python version: 3.7.3
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0            :  1.3.0 (OK)
chardet >=2.0.0                 :  3.0.4 (OK)
cloudpickle >=0.5.0             :  1.0.0 (OK)
diff_match_patch >=20181111     :  20200713 (OK)
intervaltree >=3.0.2            :  3.1.0 (OK)
IPython >=7.6.0                 :  7.20.0 (OK)
jedi =0.17.2                    :  0.17.2 (OK)
jsonschema >=3.2.0              :  3.2.0 (OK)
keyring >=17.0.0                :  18.0.0 (OK)
nbconvert >=4.0                 :  5.5.0 (OK)
numpydoc >=0.6.0                :  0.9.1 (OK)
paramiko >=2.4.0                :  2.7.2 (OK)
parso =0.7.0                    :  0.7.0 (OK)
pexpect >=4.4.0                 :  4.8.0 (OK)
pickleshare >=0.4               :  0.7.5 (OK)
psutil >=5.3                    :  5.6.2 (OK)
pygments >=2.0                  :  2.4.0 (OK)
pylint >=1.0                    :  2.3.1 (OK)
pyls >=0.36.2;<1.0.0            :  0.36.2 (OK)
pyls_black >=0.4.6              :  0.4.6 (OK)
pyls_spyder >=0.3.2             :  0.3.2 (OK)
qdarkstyle >=2.8                :  2.8.1 (OK)
qtawesome >=0.5.7               :  0.5.7 (OK)
qtconsole >=5.0.1               :  5.0.2 (OK)
qtpy >=1.5.0                    :  1.7.1 (OK)
rtree >=0.8.3                   :  0.9.7 (OK)
setuptools >=39.0.0             :  41.0.1 (OK)
sphinx >=0.6.6                  :  2.0.1 (OK)
spyder_kernels >=1.10.2;<1.11.0 :  1.10.2 (OK)
textdistance >=4.2.0            :  4.2.1 (OK)
three_merge >=0.1.1             :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0        :  1.0.2 (OK)
zmq >=17                        :  18.0.0 (OK)

# Optional:
cython >=0.21                   :  0.29.7 (OK)
matplotlib >=2.0.0              :  3.0.3 (OK)
numpy >=1.7                     :  1.16.3 (OK)
pandas >=1.1.1                  :  0.24.2 (NOK)
scipy >=0.17.0                  :  1.2.1 (OK)
sympy >=0.7.3                   :  1.4 (OK)

# Spyder plugins:
spyder_notebook                 :  0.3.2 (OK)
```

**Comment by ccordoba12** [comment]
Hey @Abdullah-B, thanks for reporting, We'll fix this in our next version (4.2.3) to be released next week.

**Comment by ccordoba12** [comment]
Sorry, we changed our mind and we're going to release 4.2.3 with a fix for this because a lot of people is finding this bug.
